Web/Edubase.Web.UI/Areas/Groups/Models/CreateEdit/CreateAcademyTrustViewModel.cs

```diff
@@ -47,7 +47,8 @@ public CreateAcademyTrustViewModel(CompanyProfile companyProfile, IEnumerable<Se
         [Required(ErrorMessage = "Please select the trust type"), DisplayName("Trust type")]
         public int? TypeId { get; set; }
 
-        [Required(ErrorMessage = "Please enter a group ID"), DisplayName("Group ID")]
+        //[Required(ErrorMessage = "Please enter a group ID"), DisplayName("Group ID")]
+        //added validation to the controller as this now depends on if the group is 16-19 secure
```

Comment:
remove comment

Reply:
sorry are you saying to remove this comment?